vast/CMakeLists.txt

```diff
@@ -87,6 +87,39 @@ elseif (${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
     COMPONENT Runtime)
 endif ()
 
+# -- cli tests -----------------------------------------------------------------
+
+add_custom_target(cli-test)
+
+add_custom_target(
+  cli-test-core
+  COMMAND
+    ${CMAKE_COMMAND} -E env
+    PATH="$<TARGET_FILE_DIR:vast::tenzir>:$ENV{PATH}:${CMAKE_CURRENT_SOURCE_DIR}/cli-test/bats/bin"
+    bats "-T" "${CMAKE_CURRENT_SOURCE_DIR}/cli-test/tests"
+  COMMENT "Executing CLI tests..."
+  USES_TERMINAL)
+
+add_dependencies(cli-test-core tenzir)
+add_dependencies(cli-test cli-test-core)
```

Comment:
More of a question than a request: Do you think it'd be possible to call `add_test` for every individual test suite? That way, we wouldn't need the separate target at all.

Reply:
I don't think this is trivial, so I would like to avoid it in the context of this PR. Maybe as a follow up.

Reply:
I don't think the value-for-money of this integration is high.
TBH I think it would be a better idea to just call `add_test` for each unit test binary and each functional test folder.
Let's discuss the pros and cons before jumping into action.